scripts/countdown.js

```diff
@@ -0,0 +1,41 @@
+function getTimeRemaining(endtime) {
+  var t = Date.parse(endtime) - Date.parse(new Date());
+  var seconds = Math.floor((t / 1000) % 60);
```

Comment:
No es muy recomendable usar var. https://hackernoon.com/why-you-shouldnt-use-var-anymore-f109a58b9b70

Reply:
Awesome!